mtags/src/main/scala-2/scala/meta/internal/mtags/Semanticdbs.scala

```diff
@@ -61,7 +61,7 @@ object Semanticdbs {
   ): TextDocumentLookup = {
     val reluri = scalaRelativePath.toURI(false).toString
     val sdocs = loadTextDocuments(semanticdbPath)
-    sdocs.documents.find(_.uri == reluri) match {
+    sdocs.documents.find(_.uri.replace("\\", "/") == reluri) match {
```

Comment:
Normally path was done `/src/main/scala/Main.scala`, but when Dotty is generating the file it's `src\main\scala\Main.scala`

Reply:
So this change this would only matter if we are looking for semantic information that dotty produced while the user is still in a scala-2 workspace right?

Reply:
This sounds like a bug in Dotty, the `uri` field should always return a URI. I think it's OK to do this replace on the Metals side just-in-case but we should make sure this is also fixed upstream in Dotty.

Reply:
Sure, we plan to follow up with a change to Dotty itself too.

Reply:
@ckipp01 This only is an issue if someone compiles Dotty code on Windows, it doesn't influence Scala-2 codebases at all.. We will not use the semanticdb information due to the relative uri mismatch